site/src/pages/WorkspacePage/Workspace.tsx

```diff
@@ -148,6 +145,29 @@ export const Workspace: FC<React.PropsWithChildren<WorkspaceProps>> = ({
   const transitionStats =
     template !== undefined ? ActiveTransition(template, workspace) : undefined;
 
+  const sidebarOption = useTab("sidebar", "");
+  const setSidebarOption = (newOption: string) => {
+    const { set, value } = sidebarOption;
+    if (value === newOption) {
+      set("");
+    } else {
+      set(newOption);
+    }
+  };
+
+  const selectedResourceId = useTab("resources", "");
+  const resources = workspace.latest_build.resources.sort(
```

Comment:
Accidental mutation here

Reply:
Not sure if I got it 🤔

Reply:
So, the `.sort` method is mutating the underlying array, even though that array is globally available throughout the app through the query cache. That could cause bugs for any other components that need the data in its original form

I don't think the `.toSorted` method is supported enough to swap that in, but you should be able to rewrite this as
```ts
 const resources = [...workspace.latest_build.resources].sort()
```